chapters/10_chapter_9.tex

```diff
@@ -0,0 +1,329 @@
+\chapter{Fault Attacks} \label{c9_ninthchapter:cha}
```

Comment:
Please use the convention chap:c9_.... for label

Reply:
From what I see in other chapters, this matches the convention.
Could you write the full convention?
Also I would recommend fixing this across the other chapters...